bf2fc6cc711aee1a0c2a/kas-fleet-manager
# Fix to properly clean clients created in mass sso

## Description
At the moment when running OCM_ENV=integration make test/integration KFM creates clients in mas-sso which dont properly get cleaned up.

Jira: https://issues.redhat.com/browse/MGDSTRM-7861

## Verification Steps
1. Launch local instance of mas-sso.By running the commands  1. make sso/setup 2.make sso/config
2. Navigate to localhost:8081.
5. Check both rhoas realm and rhoas-kafka-sre realm.
6. Run OCM_ENV=integration make test/integration .
7. You should see the clients being created during the test after refresh.
8. After test refresh the page and they should be gone.

## Checklist (Definition of Done)
